tools/reduce.py

```diff
@@ -46,7 +46,7 @@ def runtool():
             return True
     elif p.returncode == 0:
         out = comm[0] + '\n' + comm[1]
-        if ('error:' not in out) and (out.find(EXPECTED) > 0):
+        if 'error:' not in out and EXPECTED in out:
```

Comment:
It seems to me that this line is easier to read without parenthesis.
For me, wrapping single statement in parenthesis is too much and I think it's not necessary here (Python tends to be readable by humans).

Correct me, if I'm wrong.